### ISSUE TITLE
If you uninstall the system Tor, Ricochet doesn't recover to notice it's gone

### ISSUE THREAD

**Issue by armadev** [ricochet-im/ricochet#271]
Step one, install the Tor deb, then run RIcochet for the first time. It will notice the system Tor, and use it, and everything is fine.

Step two, uninstall the Tor deb. Run Ricochet again. It will just tell you "Connecting...", and it will make the little moving line gesture that indicates it's in the middle of something. If you click preferences, and go to the Tor tab, it will say "Error: No such file or directory". It pretty clearly isn't going to make any more progress, but it also isn't planning to tell me this.

**Comment by armadev** [comment]
Indeed, if I mv away the config/tor/torrc file, then Ricochet tells me on startup that "No such file or directory."

**Comment by special** [comment]
So, the first-time tor setup is recognizing this situation, but the path for existing configurations is not.
